### DISCUSSION TITLE
Receiving data from telegram

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#3904] [Q&A]
### Question

I use flet to create a telegram web app and I need to transfer data from telegram to my application, at least user_id, but I don’t understand how to do this. transmitting data in a web string as a path may not be secure

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Will be nice if you could outline how to run a simple app on telegram.
